PatreonClient.Tests/IdentityEndpointParsingTests.cs

```diff
@@ -57,7 +57,7 @@ private static PatreonHttpClient CreateClient(object data)
                 BaseAddress = new Uri("https://www.test.com/")
             };
 
-            return new PatreonHttpClient(client);
+            return new PatreonHttpClient(client, null, string.Empty);
```

Comment:
I think this will throw now because of the `null||whitespace` check

Reply:
Good catch, fixed in  44d4d16